### DISCUSSION TITLE
No 0.25 Chia per block caught

### DISCUSSION THREAD

**Discussion by serg3140** [Chia-Network/chia-blockchain#8235] [Support]
I caught block 733858.
You can see this in Spacepool.
Explorer shows 0.25 chia sent to my address
But they are not in the wallet.
What's with the wallet?
![image](https://user-images.githubusercontent.com/37830229/131103651-2bfe5d82-995a-4f01-a21d-f57153919d5a.png)
![image](https://user-images.githubusercontent.com/37830229/131103672-57b9d38e-ca59-4678-a39d-13d2a19384c4.png)
![image](https://user-images.githubusercontent.com/37830229/131103691-2e2fc49a-a98c-40f4-afae-d82d2ac63c7c.png)
![image](https://user-images.githubusercontent.com/37830229/131103719-4d0b7a15-b27a-48f9-b0c3-139cd38e7a2e.png)

**Comment by loppefaaret** [comment]
linked to issue #8233

**Comment by BrandtH22** [comment]
@serg3140 , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
